master/master.cfg

```diff
@@ -672,6 +682,70 @@ def add_halide_cmake_build_steps(factory, builder_type, configs):
                          command=get_cmake_build_command(builder_type, config, build_dir)))
 
 
+def add_halide_cmake_package_steps(factory, builder_type, configs):
+    for config in configs:
+        assert config == 'Release'
+
+        source_dir = get_halide_source_path()
+
+        # TODO: this is clumsy; we assume an integer version for llvm
+        # and also assume the Halide version matches; this is sorta
+        # true now (for Halide trunk) but needs more sophistication
+        # once we make Halide release branches
+        version = '%s.0.0' % str(_TO_VERSION[builder_type.llvm_branch])
+        target = builder_type.halide_target()
+        ext = 'zip' if builder_type.os == 'windows' else 'tar.gz'
+        pkg_name = 'Halide-%s-%s.%s' % (version, target, ext)
+
+        env = get_env(builder_type)
+        env['Clang_DIR'] = get_llvm_install_path(config, 'lib/cmake/clang')
+        env['LLVM_DIR'] = get_llvm_install_path(config, 'lib/cmake/llvm')
+        env['Halide_VERSION'] = version
+
+        if builder_type.os == 'windows':
+            # TODO: on Windows, we can't use Ninja for packaging (as we do everywhere
+            # else in this cfg) due to a bug in CMake 3.18, so we must use MSBuild;
+            # that means we must use a different build directory entirely. To simplify the
+            # world, we make this a subdir of the real build dir (so it gets cleaned properly).
```

Comment:
Same deal here... open an issue. I'll file a bug with the CMake folks; assign it to me.